pages/blog/conference-2022.md

```diff
@@ -0,0 +1,183 @@
+
+## The challenges 
+
+Because AsycnAPI is a community-first organization, I was able to follow along with the whole process of planning the conference, and I could see some of the challenges the community faced when planning to organize the conference. 
+
+Here are a few of the challenges:
+
+- **Went from hybrid to online**: The event was supposed to be an in-person and virtual event held in Barcelona, Spain. But because the foundation they partnered with couldn't find a venue within the provided date, they had to scrap the in-person idea.
+- **Fewer Sponsors**: I believe the whole planning for the 2022 AsyncAPI Conference started quite late, which made it difficult to get a budget for the event in such a short time.
+
+But despite these challenges, AsyncAPI still reached the top of the mountain by giving a blast conference with the power of the community. 
+
+## How was it done
+
+AsyncAPI proved how much you can do when you have an amazing and strong community behind you.
+
+Despite AsyncAPI leveraging their experience hosting online events, they also created a conference support squad where so many amazing folks from the community volunteered, including me. This squad involves two categories of volunteers: on-stage and off-stage. 
+
+This approach worked like a charm, bringing over 15 members of the community to help coordinate the conference in a collaborative way which, in my opinion, was what made the conference a blast!
+
+## Was it worth it? 
+
+I wouldn't say it was amazing if it weren't worth it! The conference was amazing thanks to the contributions from the organization, community, speakers, and sponsors.
+
+Look at the number of views we had on Youtube alone; don't forget numbers don't lie.
+
+- Day 1: 584 views
+- Day 2: 325 views
+- Day 3: 247 views
+
+But hold on, we celebrated too early because two weeks after the conference, the number of views doubled.
+
+- Day 1 views grew to 1403
+- Day 2 views grew to 1181
+- Day 3 views grew to 400
+
+AsyncAPI also had significant women speakers & stream hosts. Isn't this an important move in the right direction for true diversity?
+
+## Meet the heroes 
+
+With the help of these amazing organizations and individuals, AsyncAPI was able to host an incredible conference, and here's a special shoutout to each and every one of them.
+
+### Meet the Sponsors
+Platinum Sponsor
+<a href='https://www.postman.com/' target='_blank'>
+<img src='/img/sponsors/postman.png' alt='postman' width='500px' /> 
+</a>
+
+Gold Sponsor
+
+<a href='https://ably.com/' target='_blank'>
+<img src='/img/sponsors/ably.png' alt='ably' width='400px' />
+</a>
+
+
+### Media Support
+
+Special thanks to Restream for supporting us with their amazing product for media coverage.
+<a href='https://restream.io/' target='_blank'>
+<img src='/img/sponsors/restream.png' alt='restream' width='200px' />
+</a>
+
+### Meet the volunteers
```

Comment:
@Barbanio can you provide @AceTheCreator the list, or who should be and not be on it? I think Jeremy in the end did not help running it. So best would be to make sure this list is aligned with the list that was used to send swags as thank you

Reply:
Yes, of course, this is the list:

- Aishat Muibudeen 
- Alejandra Quetzalli
- Azeez Elegbede 
- Barbaño González
- Ben Granados Matarranz
- Dale Lane
- Deepak Verma
- Fran Méndez
- Karuna Tata
- Khuda Dad Nomani
- Lukasz Gornicki
- Maciej Urbanczyk
- Missy Turco
- Sergio Moya
- Souvik De
- V Thulisile Sibanda
- Wee Prods. (María and Josefo)